db/migrate/20190716121043_create_users.rb

```diff
@@ -0,0 +1,10 @@
+class CreateUsers < ActiveRecord::Migration[5.2]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :email
```

Comment:
@shaniajubelag
Let's not include this yet, since this is not related to this task.